index.html

```diff
@@ -15,5 +15,47 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <header> 
+        <img class=logo src="/img/karma-logo.svg" alt="logo">
+        <nav>
+            <ul><a href="#"><strong>Meet Karma</strong></a></ul>
+            <ul><a href="#">How it Works</a></ul>
+            <ul><a href="#">Store</a></ul>
+            <ul><a href="#">Blog</a></ul>
+            <ul><a href="#">Help</a></ul>
+            <ul><a href="#">Login</a></ul>
+        </nav>
+    </header>
+    <section class="hero">
+        <h1>Introducing Karma</h1>
+        <p>Bring Wifi with you, everywhere you go.</p>
+        <button>Learn More</button>
+    </section>
+    <h2> Everyone needs a little Karma.</h2>
+    <section class="articlecontainer">
+        <article>
+            <img src="/img/icon-devices.svg" alt="different devices with different screensizes">
+            <p>Internet for all devices</p>
+        </article>
+        <article>
+            <img src="/img/icon-coffee.svg" alt="a cup with steam">
+            <p>Boost your Productivity</p>
+        </article>
+        <article>
+            <img src="/img/icon-refill.svg" alt="a petrol tank">
```

Comment:
I like your sense of humour there for your alt="Petrol tank", why didn't you use something like "Device Icon"?